net/message/message.go

```diff
@@ -50,35 +50,27 @@ func DecodeMessage(b []byte) (Message, error) {
 	if len(b) == 0 {
 		return nil, errors.New("invalid message bytes")
 	}
+	msgType := Type(b[0])
+	msgJSON := b[1:]
+	var msg Message
```

Comment:
check len(b) > 1 instead?

Reply:
The above code is safe.  We really only need to check that `len(b)` is non-zero before referencing `b[0]`.  If `b[1:]` is empty, that will get caught by `json.Unmarshal`.  The Unmarshal will definitely fail if we don't have at least `{}`.  I'll update it to `< 3`.